ChatApp/ChatView/ChatViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  ChatViewController.swift
+//  ChatApp
+//
+//  Created by Giorgi Makadze on 18.04.2023.
+//
+
+import UIKit
+
+class ChatViewController: UIViewController {
+    
+    let modeButton = UIButton(type: .custom)
+    
+    let lightModeButton = UIButton(type: .custom)
+    let darkModeButton = UIButton(type: .custom)
+    
+    let senderMessageView = MessageView()
+    let receiverMessageView = MessageView()
+    let separator = UIView()
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        
+        separator.backgroundColor = .yellow
+        
+        //view.backgroundColor = .yellow
+        let stackView = UIStackView(arrangedSubviews: [senderMessageView, receiverMessageView])
+        stackView.axis = .vertical
+        stackView.spacing = 0 // gasasworebeli
+        stackView.distribution = .fillEqually
+        
+        view.addSubview(stackView)
+        view.addSubview(separator)
+        
+        separator.translatesAutoresizingMaskIntoConstraints = false
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            
+            //            separator.heightAnchor.constraint(equalToConstant: 100),
+            stackView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            stackView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor),
+            stackView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            stackView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            
+            separator.topAnchor.constraint(equalTo: senderMessageView.bottomAnchor),
+            separator.bottomAnchor.constraint(equalTo: receiverMessageView.topAnchor),
```

Comment:
you should not setup components directly in viewDidLoad. creating separate methods and calling them here would be better. it will help you to easily navigate and add some functionality since you will have more organized code.